docs/examples/PanelCollapsible.js

```diff
@@ -12,7 +12,11 @@ class Example extends React.Component {
         <Button onClick={ ()=> this.setState({ open: !this.state.open })}>
           click
         </Button>
-        <Panel collapsible expanded={this.state.open}>
+        <Panel
+          collapsible
+          id="collapsible-panel-example"
+          expanded={this.state.open}
+        >
           Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
```

Comment:
I would keep both `<Panel.Collapse>` and `<Panel.Body>`, but make `<Panel.Body collapse>` just be shorthand for a single body element with `<Panel.Collapse>` around it.

It is having two ways to do the same thing, but I think it's a pragmatic compromise.